Azure/iotedge
# [k8s] Add pod security context to create options

Provide an option for user to specify pod security context.
To verify changes:
```
1. install edge on Kubernetes
2. deploy a module with default parameters and default image.
3. verify the pod is up and running
4. apply the pod security context as a createOptions section on a portal
5. wait until agent reconcile changes
6. check new pod for the deployment is in a failed state.
```
